src/components/TodoList/TodoList.tsx

```diff
@@ -2,223 +1,96 @@
 import React from 'react';
+import { useAppSelector } from '../../app/store';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
```

Comment:
Verify that the `setCurrentTodo` action is correctly defined in the `../../features/currentTodo` module. It should be an action creator that updates the current todo in the Redux store.